scripts/release_scripts/gcloud_adapter.py

```diff
@@ -134,6 +134,10 @@ def switch_version(app_name, version_to_switch_to):
         common.GCLOUD_PATH, 'app', 'services', 'set-traffic',
         'default', '--splits', '%s=1' % version_to_switch_to,
         '--project=%s' % app_name])
+    subprocess.check_output([
+        common.GCLOUD_PATH, 'app', 'services', 'set-traffic',
+        'cloud-datastore-admin', '--splits', '%s=1' % version_to_switch_to,
```

Comment:
Will this actually be the same version as for the main app? (I'm not sure.)

Reply:
Yes, it should be to avoid unnecessary switches as we discussed in our meeting.

Reply:
No, what I mean is, are you sure you have got the version name for the second app correct? I don't see where it gets specified -- these are two different apps. In general they don't share the same version name, I believe.

Reply:
Looking into this, will update in some time.

Reply:
@seanlip I am not able to find any specific way on how the version number is decided for cloud-datastore-admin. Is it a good idea to open the versions tab in browser and ask the release co-ordinator to provide the latest version number from there?

Reply:
Maybe you could just get the list of versions automatically and pick the latest one?

Reply:
Sounds good, done.